galsim/base.py

```diff
@@ -1247,8 +1237,8 @@ class DeVaucouleurs(GSObject):
 
     def __init__(self, half_light_radius=None, scale_radius=None, flux=1., trunc=0.,
                  flux_untruncated=False, gsparams=None):
-        GSObject.__init__(self, _galsim.SBDeVaucouleurs(scale_radius, half_light_radius, flux,
-                                                        trunc, flux_untruncated, gsparams))
+        GSObject.__init__(self, _galsim.SBSersic(4, scale_radius, half_light_radius, flux,
```

Comment:
Weird, my first (emailed) reply to this showed up on GitHub as a reply to the whole issue rather than associated with this comment.  So I deleted it and am trying again.  All this was to say... it's all good.